app.py

```diff
@@ -8,7 +8,7 @@ def chat(user_input):
     try:
         response = message_to_bot(user_input, clf, learn_response)
     except:
-        return jsonify({'message': "Unable to get response"}, 500)
+        return jsonify({'message': ('Unable to get response', 0)}, 500)
```

Comment:
Can you clarify the motivation behind this change?

Reply:
The response from the mapbot is a tuple **(message, learn_response)**. So I changed the error response also to a tuple. Because if someone integrates the API he will then have to write two different statements for extracting the response (one for the normal response and the other for error response if any).

Reply:
The change should be available in all the responses